drivers/include/net/netdev.h

```diff
@@ -378,14 +392,22 @@ typedef struct netdev_driver {
      * @brief   Set an option value for a given network device
      *
      * @pre `(dev != NULL)`
+     * @pre for scalar types of @ref netopt_t `max_len` must be of exactly that
+     *      length (see [netopt documentation](@ref net_netopt) for type)
+     * @pre for array types of @ref netopt_t `max_len` must lesser or equal the
+     *      required length (see [netopt documentation](@ref net_netopt) for
+     *      type)
      *
      * @param[in] dev       network device descriptor
      * @param[in] opt       option type
      * @param[in] value     value to set
      * @param[in] value_len the length of @p value
      *
-     * @return              number of bytes used from @p value
-     * @return              `< 0` on error, 0 on success
+     * @return              number of bytes written to @p value
+     * @return              `-ENOTSUP` if @p opt is not configurable for the
+     *                      device
+     * @return              `-EINVAL` if @p value is invalid an invalid value
+     *                      for @p opt
```

Comment:
This sentence doesn't parse for me.

Reply:
For me neither. Only drank weak coffee before that so that might be the cause ;-).